leverage/modules/project.py

```diff
@@ -279,6 +280,16 @@ def load_project_config():
         raise Exit(1)
 
 
+def validate_config(config: dict):
+    """
+    Run a battery of validations over the config file (project.yaml).
+    """
+    if not re.match(r"^[a-z0-9]([a-z0-9]|-){1,23}[a-z0-9]$", config["project_name"]):
+        raise ExitError(1, "Project name is not valid. Only lower case alphanumerics characters are allowed.")
```

Comment:
```suggestion
        raise ExitError(1, "Project name is not valid. Only lower case alphanumeric characters are allowed.")
```
Just a small grammar correction!